agent/06_agent_create_cluster.sh

```diff
@@ -294,6 +327,14 @@ function agent_pxe_boot() {
       done
 }
 
+function create_appliance() {
+    local asset_dir="$(realpath "${1}")"
+
+    # Build appliance with `debug-base-ignition` flag for using the custom openshift-install
+    # binary from assets directory.
+    sudo podman run -it --rm --privileged --net=host -v ${asset_dir}:/assets:Z ${APPLIANCE_IMAGE} build --debug-base-ignition
```

Comment:
```suggestion
    sudo podman run -it --rm --pull newer --privileged --net=host -v ${asset_dir}:/assets:Z ${APPLIANCE_IMAGE} build --debug-base-ignition
```
Can we have podman pull a newer version of one is available?

Reply:
Yeah, good idea, added.